MyToolsYourToolsClient/src/app/services/notification.service.ts

```diff
@@ -2,12 +2,17 @@ import { Injectable } from '@angular/core';
 import { Notification } from '../models/Notification';
 import { NotificationType } from '../enums/NotificationType';
 import { Observable, of } from 'rxjs';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+};
 
 @Injectable({
   providedIn: 'root'
 })
 export class NotificationService {
+  baseUrl = 'https://localhost:5001/api/notifications/';
```

Comment:
to samo co wyzej

Reply:
zmieniam bo mi nie działa, robi mi api na 5001. To chyba przez vsCode-a